server/fridgeServer.go

```diff
@@ -0,0 +1,76 @@
+package server
+
+import (
+	"sync"
+
+	"github.com/ASeegull/SmartFridge/server/config"
+	"github.com/ASeegull/SmartFridge/server/database"
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	defaultPort            = "9000"
+	defaultHost            = "localhost"
+	defaultReadBufferSize  = 1024
+	defaultWriteBufferSize = 1024
+	defaultWebsocketSleep  = 2
+)
+
+var serverConfig *config.ServerConfig
+var wg *sync.WaitGroup
+
+//SetWaitGroup sets WaitGroup
+func SetWaitGroup(waitGroup *sync.WaitGroup) {
+	wg = waitGroup
+}
+
+//ReadConfig reads config from file
+func ReadConfig() {
+	var mongoDBConfig *config.MongoConfig
+	if err := config.ReadConfig(); err != nil {
+		serverConfig = &config.ServerConfig{
+			Port:            defaultPort,
+			Host:            defaultHost,
+			ReadBufferSize:  defaultReadBufferSize,
+			WriteBufferSize: defaultWriteBufferSize,
+			WebsocketSleep:  defaultWebsocketSleep}
+
+		log.Println("cannot read config. Used default values")
+	} else {
+		mongoDBConfig = config.GetMongoConfig()
+		serverConfig = config.GetServerConfig()
+	}
+
+	setUpgrader()
+	database.InitiateMongoDB(mongoDBConfig)
```

Comment:
Why are you initialize mongo in `ReadConfig` ? Please move this logic to database package